README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
+| `customHttpHeadersProvider`                | Add a custom HTTP header provider. Useful for [dynamic custom HTTP headers](#dynamic-custom-http-headers).                                                          | No       | `null`                               |
+| `disableMetrics`                           | A boolean indicating whether the client should disable sending usage metrics to the Unleash server.                                                                 | No       | `false`                              |
+| `enableProxyAuthenticationByJvmProperties` | Enable support for [using JVM properties for HTTP proxy authentication](#http-proxy-with-authentication).                                                           | No       | `false`                              |
+| `environment`                              | The name of the current environment.                                                                                                                                | No       | `null`                               |
+| `fallbackStrategy`                         | ???                                                                                                                                                                 | No       | `null`                               |
+| `fetchTogglesInterval`                     | How often (in seconds) the client should check for toggle updates.                                                                                                  | No       | `10`                                 |
+| `instanceId`                               | An identifier for your instance. Typically a hostname, pod id or something similar. Unleash uses this internally for ...something                                   | Yes      | `null`                               |
+| `namePrefix`                               | ???                                                                                                                                                                 | No       |                                      |
+| `projectName`                              | The name of the Unleash project?                                                                                                                                    | No       |                                      |
+| `proxy`                                    | ???                                                                                                                                                                 | No       |                                      |
+| `scheduledExecutor`                        | ???                                                                                                                                                                 | No       |                                      |
```

Comment:
What does this do?

Reply:
This beastie controls the timing and running of tasks (mostly for fetching Toggles), there's a default threaded implementation called `UnleashScheduledExecutorImpl`, people who know this code better than me will have to clue me in on when/why you'd want your own custom implementation here though